libs/designer/src/lib/ui/Designer.tsx

```diff
@@ -134,7 +134,8 @@ export const Designer = (props: DesignerProps) => {
   const isFetchingInitialData = useIsFetching({
     predicate: (query) => {
       const queryKeysToWatch = ['connections', 'manifest', 'apiWithSwaggers'];
-      return (query.queryKey as string[]).some((val) => queryKeysToWatch.includes(val));
+      const queryKeyArray = query.queryKey && isString(query.queryKey) ? [query.queryKey] : (query.queryKey as string[]);
```

Comment:
you can remove this, I have another PR fixing the same issue already haha

Reply:
Alright, thanks!!